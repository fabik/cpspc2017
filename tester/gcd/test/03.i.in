100
496 486 480 496 488 484 483 498 492 495 480 490 488 490 493 483 493 497 493 483 481 486 480 482 482 497 488 489 481 490 494 498 486 485 489 483 494 491 485 482 498 493 485 500 495 488 497 488 483 487 482 495 483 485 490 499 485 497 495 492 481 484 499 500 495 486 487 486 493 494 494 488 494 498 489 483 496 486 494 495 496 483 485 500 489 491 480 480 483 489 484 481 494 492 499 500 481 486 491 487
