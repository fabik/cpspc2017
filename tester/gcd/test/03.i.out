1
501
1001
1501
2001
2500
2999
3498
3996
4494
4992
5490
5987
6484
6981
7478
7974
8470
8966
9462
10290
10790
11290
11790
12289
12788
13286
13784
14282
14779
15276
15773
16269
16765
17261
17756
18251
18746
19241
19736
20230
20724
21218
21712
22206
22700
23193
23686
24179
24672
25164
25656
26147
26638
27128
27618
28108
28597
29086
29575
30064
30552
31040
31528
32016
32504
32991
33478
33964
34450
34936
35422
35908
36394
36879
37364
37849
38334
38819
39303
39787
40270
40753
41236
41719
42202
42685
43168
43651
44133
44615
45097
45578
46059
46540
47021
47501
47981
48461
48941
